src/vunnel/providers/rhel/csaf_client.py

```diff
@@ -54,15 +54,17 @@ def __init__(
         self,
         workspace: Workspace,
         logger: logging.Logger,
+        skip_download: bool,
+        max_workers: int,
         latest_url: str = ADVISORIES_LATEST_URL,
-        skip_download: bool = False,
```

Comment:
Does this make an impact where this no longer has the default of `False` and we rely on the config being required to be passed?